pwndbg/gdblib/kernel/__init__.py

```diff
@@ -76,6 +78,21 @@ def kversion() -> str:
     return pwndbg.gdblib.memory.string(version_addr).decode("ascii").strip()
 
 
+@requires_debug_syms()
+@pwndbg.lib.memoize.reset_on_start
+def krelease() -> Tuple[int, ...]:
+    # try to extract (major.minor.patch) version
+    match = re.search(r"Linux version (\d+)\.(\d+)\.(\d+)", kversion())
+    if match:
+        return tuple(map(int, match.groups()))
+    # try to extract (major.minor) version and append 0 for patch version
+    match = re.search(r"Linux version (\d+)\.(\d+)", kversion())
+    if match:
+        return tuple(map(int, match.groups())) + (0,)
+    # not found
```

Comment:
I don't think this comment is particularly helpful 😄

Reply:
i removed the comments :)